libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestAPI.m

```diff
@@ -49,6 +50,14 @@ @implementation SFRestAPI
 @synthesize activeRequests=_activeRequests;
 @synthesize currentNetwork=_currentNetwork;
 
+__strong static NSDateFormatter *httpDateFormatter;
```

Comment:
It's good practice to initialize `static` member variables to `nil` in the assignment, in some rare race conditions where it can be dereferenced before being allocated.